Documentation/observability/metrics.rst

```diff
@@ -592,6 +602,24 @@ Name                                     Labels
 ``lbipam_services_unsatisfied_total``                                                                      Enabled    Number of services which did not get requested IPs
 ======================================== ================================================================= ========== ========================================================
 
+Controllers
+~~~~~~~~~~~
+
+======================================== ================================================== ========== ========================================================
+Name                                     Labels                                             Default    Description
+======================================== ================================================== ========== ========================================================
+``controllers_group_runs_total``         ``status`` ``group_name``                          Enabled    Number of times that a controller process was run, labeled by controller group name (configurable)
+======================================== ================================================== ========== ========================================================
+
+The ``controllers_group_runs_total`` metric reports the success
+and failure count of each controller within the system, labeled by
+controller group name and completion status. Enabling this metric is
+on a per-controller basis. This is configured using an allow-list
+which is passed as the ``controller-group-metrics`` configuration
+option. The current default set for ``cilium-operator`` found in
+the Cilium Helm chart is "all", which enables the metric for all
+controller groups. The special names "all" and "none" are supported.
```

Comment:
Can you reword this last sentence a little bit to help introduce that "all" is a special name before describing it as the default value? I was thinking of something like this:

```suggestion
the Cilium Helm chart is the special name "all", which enables the
metric for all controller groups. The special name "none" is also
supported.
```

[2]